### PR TITLE
Add Join as a notification option

### PR DESCRIPTION
**Pull request by jkluch**
Join can push notifications to android devices.

When making this I had to parse out the url from the message sent to notifications due to how join works.
I'm questioning though if it even makes sense to send the url in a notification though.  It's a one time use url right?  And it's immediately being used on the local machine.  I tried opening the url on my phone and it didn't work.  Maybe I'm just misunderstanding what's required for the nvidia shopping cart url.

### CODE REVIEW

File: notifications/notifications.py [hunk 1]
```diff
@@ -46,6 +50,8 @@ def send_notification(self, message):
                 executor.submit(self.twilio_handler.send, message)
             if self.discord_handler.enabled:
                 executor.submit(self.discord_handler.send, message)
+            if self.join_handler.enabled:
+                self.join_handler.send(message)
```
Comment (Hari-Nagarajan) [inline notifications/notifications.py:54]: Can you have this follow the executor pattern?

**Comment by Hari-Nagarajan** [comment]
Could you also add some information about this to the readme?

**Comment by Hari-Nagarajan** [comment]
The URL issue is only with nvidia, but the notification handler is used for multiple stores, so it should still be ok. The URL issue is something we're looking into.

**Review by Hari-Nagarajan — APPROVED** [review]
Thanks! LGTM

**Review by WillsB3 — APPROVED** [review]

**Pull request merged**